specutils/io/default_loaders/sixdfgs_reader.py

```diff
@@ -0,0 +1,54 @@
+import astropy.io.fits as fits
+from astropy.table import QTable
+from astropy.units import Unit
+from specutils.io.registers import data_loader
+from specutils import Spectrum1D
+
+SIXDFGS_PRIMARY_HDU_KEYWORDS = ["OBSRA", "OBSDEC", "Z", "Z_HELIO", "QUALITY"]
+COUNTS_PER_SECOND = Unit("counts/s", parse_strict="silent")
+
+
+def identify_6dfgs_tabular_fits(origin, *args, **kwargs):
+    """
+    Identify if the current file is a 6dFGS file (stored as a table)
+    """
+    with fits.open(origin) as hdulist:
+        if len(hdulist) != 2:
+            return False
+        primary_hdu = hdulist[0]
+        if primary_hdu.header["NAXIS"] != 0:
+            return False
+        for keyword in SIXDFGS_PRIMARY_HDU_KEYWORDS:
+            if keyword not in primary_hdu.header:
+                return False
+        return True
+
+
+@data_loader("6dFGS-tabular",
+             identifier=identify_6dfgs_tabular_fits,dtype=Spectrum1D,
+             extensions=["fit", "fits"])
+def sixdfgs_tabular_fits_loader(filename, **kwargs):
+    """
+    Load the tabular variant of a 6dFGS file
+
+    Parameters
+    ----------
+    file_name: str
+        The path to the FITS file
+
+    Returns
+    -------
+    data: Spectrum1D
+        The 6dF spectrum that is represented by the data in this table.
+    """
+    with fits.open(filename) as hdulist:
+        header = hdulist[0]
```

Comment:
Ah, yeah, typo on my part. :)

The original 6dFGS Final Data Release released the observations in two formats, the tabular one (which this loader loads), which only contained flux and wavelength (no uncertainty), and one where the reduced spectra (with sky/uncertainty information, both with the separate arms and combined) was included with 2MASS and SuperCOSMOS (I think?) postage stamps. Then GAMA came along and only used the combined and best version (so single HDU). And now the plan for Data Central is to split all the 6dFGS spectra into their own individual file (avoiding the need to keep downloading postage stamps, and make it easier to understand what data each object has).

I thought about including the header from the actual table HDU, but there's no metadata there (other than the table metadata), so rather than adding another key/dict pair, I left it out. Happy to include it for completeness sake though.

Reply:
If the table HDU header really contains no information of interest (you could also check `table.meta` below, which will only preserve non-trivial header cards), then there is probably no need to add it. #617 has raised a few more questions on how to organise the header info inside `meta`.